DomainParser.TestConsole/Program.cs

```diff
@@ -8,6 +8,7 @@ namespace DomainParser.TestConsole {
     class Program {
         static void Main(string[] args) {
 
+            //What the hell is d: ????  :) :D (*_*)
             TLDRulesCache.RulesFileLocation = @"d:\" + Path.GetFileName(TLDRulesCache.RulesFileLocation);
```

Comment:
Great catch.  Looks like a contributor accidentally hardcoded the root path for a test.... whoopsie. 

⭐️